src/utils/clone.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { clone } from './clone'
+
+describe('utils', () => {
+  describe('#clone', () => {
+    describe('for number', () => {
+      it('returns the original entry', () => {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const original = 1 as any
```

Comment:
I'm curious why you need this `any` here. Won't `clone` accept `number` as a valid type for its T?

Reply:
Right, this can be removed now 👍